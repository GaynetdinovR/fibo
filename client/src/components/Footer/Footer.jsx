import styles from "../../styles/components/Footer.module.sass";
import Logo from "../../ui/Logo.jsx";
import OvalButton from "../../ui/OvalButton.jsx";
import PhoneNumber from "../../ui/PhoneNumber.jsx";
import LinkUI from "../../ui/LinkUI.jsx";
import FooterNavBar from "./components/FooterNavBar.jsx";
import FooterContacts from "./components/FooterContacts.jsx";

const Footer = () => {
	const windowWidth = window.innerWidth;

	const logo = (
		<div className={styles.footer__logo}>
			<Logo/>
		</div>
	);

	const cardTypes = [
		{ src: "./icons/visa.png", alt: "visa" },
		{ src: "./icons/paypal.png", alt: "paypal" },
		{ src: "./icons/mastercard.png", alt: "mastercard" }
	];

	const copyright = (
		<div className={styles.footer__copyright}>
			<span>YaBao Все права защищены © 2021</span>
			<div className={styles.footer__copyright_imgs}>
				{cardTypes.map((item, i) => (
					<div key={i} className={styles.footer__copyright_img}>
						<img src={item.src} alt={item.alt} />
					</div>
				))}
			</div>
		</div>
	);

	const phoneInfo = (
		<div className={styles.footer__phone_info}>
			<PhoneNumber
				phoneNumber={"8 499 391-84-49"}
				className={styles.footer__phone_number}
			/>

			<OvalButton className={styles.footer__get_call}>
				Заказать звонок
			</OvalButton>
		</div>
	);

	const phoneMediumContainer = (
		<div className={styles.footer__phone_medium_container}>
			{logo} {phoneInfo}
		</div>
	);

	return (
		<footer className={styles.footer}>
			<div className={styles.footer__left_side}>
				{windowWidth <= 540 ? phoneMediumContainer : null}

				{windowWidth > 960 ? logo : null}

				<div className={styles.footer__info}>
					{windowWidth <= 960 && windowWidth > 540 ? logo : null}

					<LinkUI className={styles.footer__info_link} href={"#"}>
						Калорийность и состав
					</LinkUI>

					<LinkUI className={styles.footer__info_link} href={"#"}>
						Правовая информация
					</LinkUI>
				</div>

				<FooterNavBar />

				{windowWidth > 960 ? <>{copyright}</> : null}
			</div>

			<div className={styles.footer__right_side}>
				<FooterContacts />

				{windowWidth > 540 ? phoneInfo : null}

				{windowWidth <= 960 ? copyright : null}
			</div>

			<div className={styles.footer__bg_img}>
				<img src="./content/logo_2.png" alt="bg" />
			</div>
		</footer>
	);
};
export default Footer;
