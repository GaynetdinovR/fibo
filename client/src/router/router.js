import { createBrowserRouter } from "react-router-dom";
import HomePage from "./pages/HomePage.jsx";
import Template from "../ui/Template.jsx";
import ContactsPage from "./pages/ContactsPage.jsx";
import PromoPage from "./pages/PromoPage.jsx";

const router = createBrowserRouter([
	{
		path: "/",
		element: (
			<Template>
				<HomePage />
			</Template>
		)
	},
	{
		path: "contacts",
		element: (
			<Template>
				<ContactsPage />
			</Template>
		)
	},
	{
		path: "promo",
		element: (
			<Template>
				<PromoPage />
			</Template>
		)
	}
]);

export default router;
