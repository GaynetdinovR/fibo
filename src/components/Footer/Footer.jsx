import styles from "../../styles/components/Footer.module.sass";
import Logo from "../../ui/Logo.jsx";
import LinkInList from "../../ui/LinkInList.jsx";
import OvalButton from "../../ui/OvalButton.jsx";
import PhoneNumber from "../../ui/PhoneNumber.jsx";
import Link from "../../ui/Link.jsx";
import H6 from "../../ui/H6.jsx";

const Footer = () => {
	const windowWidth = window.innerWidth;

	const links = ["YouTube", "Facebook", "Instagram", "ВКонтакте"];
	const contacts = [
		{ src: "./icons/viber.png", alt: "viber" },
		{ src: "./icons/skype.png", alt: "skype" },
		{ src: "./icons/messenger.png", alt: "messenger" },
		{ src: "./icons/telegram.png", alt: "telegram" },
		{ src: "./icons/facebook.png", alt: "facebook" },
		{ src: "./icons/vk.png", alt: "vk" }
	];

	const logo = (
		<div className={styles.footer__logo}>
			<Logo />
		</div>
	);

	const cardTypes = [
		{ src: "./icons/visa.png", alt: "visa" },
		{ src: "./icons/paypal.png", alt: "paypal" },
		{ src: "./icons/mastercard.png", alt: "mastercard" }
	];

	const copyright = (
		<div className={styles.footer__copyright}>
			<span>YaBao Все права защищены © 2021</span>
			<div className={styles.footer__copyright_imgs}>
				{cardTypes.map((item, i) => (
					<div key={i} className={styles.footer__copyright_img}>
						<img src={item.src} alt={item.alt} />
					</div>
				))}
			</div>
		</div>
	);

	const phoneInfo = (
		<div className={styles.footer__phone_info}>
			<PhoneNumber phoneNumber={"8 499 391-84-49"} className={styles.footer__phone_number} />

			<OvalButton className={styles.footer__get_call}>
				Заказать звонок
			</OvalButton>
		</div>
	);

	return (
		<footer className={styles.footer}>
			<div className={styles.footer__left_side}>
				{windowWidth <= 540 ? <div className={styles.footer__phone_medium_container}>{logo}{phoneInfo}</div> : ""}

				{windowWidth > 960 ? logo : ""}
				<div className={styles.footer__info}>
					{windowWidth <= 960 && windowWidth > 540 ? logo : ""}
					<Link className={styles.footer__info_link} href={"#"}>Калорийность и состав</Link>
					<Link className={styles.footer__info_link} href={"#"}>Правовая информация</Link>
				</div>
				<nav className={styles.footer__nav}>
					<H6 className={styles.footer__nav_title}>Мы в соцсетях</H6>
					<ul className={styles.footer__nav_list}>
						{links.map((item, i) => (
							<LinkInList key={i} liClass={styles.footer__nav_item} aClass={styles.footer__nav_link}>
								{item}
							</LinkInList>
						))}
						<li className={styles.footer__nav_item}>
							<address className={styles.footer__nav_address}>
								<span> Москва ул. Проспект </span>
								<span> Вернадского 86В </span>
							</address>
						</li>
					</ul>
				</nav>

				{windowWidth > 960 ? <>{copyright}</> : ""}
			</div>

			<div className={styles.footer__right_side}>
				<nav className={styles.footer__contacts}>
					<H6 className={styles.footer__contacts_title}>Остались вопросы? А мы всегда на связи:</H6>
					<ul className={styles.footer__contacts_list}>
						{contacts.map((item, i) => (
							<LinkInList key={i} liClass={styles.footer__contacts_item}
										aClass={styles.footer__contacts_link_img}>
								<img src={item.src} alt={item.alt} />
							</LinkInList>
						))}
						<LinkInList liClass={styles.footer__contacts_item} aClass={styles.footer__contacts_link}>
							Написать нам
						</LinkInList>
					</ul>
				</nav>

				{windowWidth > 540 ? phoneInfo : ""}

				{windowWidth <= 960 ? copyright : ""}
			</div>

			<div className={styles.footer__bg_img}>
				<img src="./content/logo_2.png" alt="bg" />
			</div>
		</footer>
	);
};
export default Footer;