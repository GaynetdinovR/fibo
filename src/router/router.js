import { createBrowserRouter } from "react-router-dom";
import HomePage from "./pages/HomePage";
import Template from "../components/Template";

const router = createBrowserRouter([
	{
		path: "/",
		element: (
			<Template>
				<HomePage />
			</Template>
		)
	}
]);

export default router;
