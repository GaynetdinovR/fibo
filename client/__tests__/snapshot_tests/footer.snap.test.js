import renderer from "react-test-renderer";
import { expect, it } from "@jest/globals";
import Footer from "../../src/components/Footer/Footer.jsx";
import { RouterProvider } from "react-router-dom";
import router from "../../src/router/router.js";

it("renders correctly", () => {
	const tree = renderer.create(<RouterProvider router={router}><Footer /></RouterProvider>).toJSON();
	expect(tree).toMatchSnapshot();
});
