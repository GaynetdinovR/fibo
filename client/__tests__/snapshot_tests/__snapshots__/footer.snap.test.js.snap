// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="notification-container notification-container-empty"
  >
    <div />
  </div>,
  <header
    className="header"
  >
    <div
      className="header__top_side"
    >
      <div
        className="header__logo"
        onClick={[Function]}
      >
        <a
          className="logo"
          href="/"
          onClick={[Function]}
        >
          <img
            alt="logo"
            src="./icons/logo_1.png"
          />
        </a>
      </div>
      <div
        className="header__left_top_side"
      >
        <div
          className="header__location"
        >
          <span>
             Доставка пасты 
          </span>
          <span
            className="header__city"
          >
             Москва 
          </span>
        </div>
        <div
          className="header__service"
        >
          <div
            className="header__service_img"
          >
            <img
              alt="yandex_food"
              src="./icons/yandex_food.png"
            />
          </div>
          <span
            className="header__service_name"
          >
            Яндекс Еда
          </span>
          <div
            className="header__service_grade"
          >
            <span>
              4.8
            </span>
            <div>
              <img
                alt="star"
                src="./icons/star.png"
              />
            </div>
          </div>
          <div
            className="header__delivery_time"
          >
            <span>
              Время доставки
            </span>
            <span>
              от 31 мин
            </span>
          </div>
        </div>
      </div>
      <div
        className="header__right_top_side"
      >
        <button
          className="header__get_call oval_btn"
        >
          Заказать звонок
        </button>
        <a
          className="header__phone_number phone_number"
          href="tel:84993918449"
        >
          8 499 391-84-49
        </a>
      </div>
    </div>
    <nav
      className="header__navbar"
    >
      <ul
        className="header__list"
        onClick={[Function]}
      >
        <li
          className="header__item link_in_list__item"
        >
          <a
            className="header__link link"
            href="#pizza"
          >
            Пиццы
          </a>
        </li>
        <li
          className="header__item link_in_list__item"
        >
          <a
            className="header__link link"
            href="#pasta"
          >
            Пасты
          </a>
        </li>
        <li
          className="header__item link_in_list__item"
        >
          <a
            className="header__link link"
            href="#soup"
          >
            Супы
          </a>
        </li>
        <li
          className="header__item link_in_list__item"
        >
          <a
            className="header__link link"
            href="#salad"
          >
            Салаты
          </a>
        </li>
        <li
          className="header__item link_in_list__item"
        >
          <a
            className="header__link link"
            href="#snack"
          >
            Закуски
          </a>
        </li>
        <li
          className="header__item link_in_list__item"
        >
          <a
            className="header__link link"
            href="#drink"
          >
            Напитки
          </a>
        </li>
        <li
          className="header__item link_in_list__item"
        >
          <a
            className="header__link link"
            href="/promo"
            onClick={[Function]}
          >
            Акции
          </a>
        </li>
        <li
          className="header__item link_in_list__item"
        >
          <a
            className="header__link link"
            href="/contacts"
            onClick={[Function]}
          >
            Контакты
          </a>
        </li>
      </ul>
      <div
        className="header__right_bottom_side"
      >
        <button
          className="header__auth"
          onClick={[Function]}
        >
          Войти
        </button>
        <button
          className="header__cart_btn default_button"
          onClick={[Function]}
        >
          Корзина | 0
        </button>
      </div>
    </nav>
  </header>,
  <div
    className="modal__none"
    onClick={[Function]}
  >
    <aside
      className="auth_modal modal"
      onClick={[Function]}
    >
      <button
        className="modal__close_btn"
        onClick={[Function]}
      >
        <img
          alt="close"
          src="./icons/cross.png"
        />
      </button>
      <div
        className="auth_modal__content"
      >
        <h4
          className="auth_modal__title h3"
        >
          Вход на сайт
        </h4>
        <label
          className="auth_modal__input_wrap"
          onClick={[Function]}
        >
          <span
            className="auth_modal__input_label"
          >
            Номер телефона
          </span>
          <div
            className="input__wrap"
          >
            <input
              className="auth_modal__input input"
              disabled={false}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onMouseDown={[Function]}
              onPaste={[Function]}
              placeholder="+7 999 999-99-99"
            />
            <div
              className="none"
            >
              <div
                className="input_error_alert__img"
              >
                <img
                  alt="warn"
                  src="./icons/warn.png"
                />
              </div>
              <span
                className="input_error_alert__text"
              >
                Неправильный вид номера
              </span>
            </div>
          </div>
        </label>
        <div
          className="auth_modal__bottom_side"
        >
          <button
            className="auth_modal__get_code_btn default_button"
            onClick={[Function]}
          >
            Выслать код
          </button>
          <p
            className="auth_modal__terms_of_use text"
          >
            <span>
              Продолжая, вы соглашаетесь 
            </span>
            <a
              href="https://en.wikipedia.org/wiki/Terms_of_service"
              target="blank"
            >
              со сбором и обработкой персональных данных и пользовательским соглашением
            </a>
          </p>
        </div>
      </div>
    </aside>
  </div>,
  <main>
    <a
      className="button_to_top"
      href="#"
    >
      <img
        alt="top_arrow"
        src="./icons/top_arrow.png"
      />
    </a>
    <div
      className="react-multi-carousel-list banners__carousel "
      dir="ltr"
    >
      <ul
        className="react-multi-carousel-track "
        onMouseDown={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseMove={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "overflow": "unset",
            "transform": "translate3d(0px,0,0)",
            "transition": "none",
          }
        }
      />
    </div>
    <aside
      className="new_products"
    >
      <h4
        className="new_products__title h4"
      >
        Новинки
      </h4>
      <div
        className="new_products__content"
      >
        <button
          className="new_products__new_product"
        >
          <div
            className="new_product__img"
          >
            <img
              alt="product"
            />
          </div>
          <div
            className="new_product__info"
          >
            <h5
              className="new_product__name h5"
            />
            <span
              className="new_product__price"
            >
               ₽
            </span>
          </div>
        </button>
        <button
          className="new_products__new_product"
        >
          <div
            className="new_product__img"
          >
            <img
              alt="product"
            />
          </div>
          <div
            className="new_product__info"
          >
            <h5
              className="new_product__name h5"
            />
            <span
              className="new_product__price"
            >
               ₽
            </span>
          </div>
        </button>
        <button
          className="new_products__new_product"
        >
          <div
            className="new_product__img"
          >
            <img
              alt="product"
            />
          </div>
          <div
            className="new_product__info"
          >
            <h5
              className="new_product__name h5"
            />
            <span
              className="new_product__price"
            >
               ₽
            </span>
          </div>
        </button>
        <button
          className="new_products__new_product"
        >
          <div
            className="new_product__img"
          >
            <img
              alt="product"
            />
          </div>
          <div
            className="new_product__info"
          >
            <h5
              className="new_product__name h5"
            />
            <span
              className="new_product__price"
            >
               ₽
            </span>
          </div>
        </button>
      </div>
    </aside>
    <section
      className="products"
    >
      <div
        className="products__categories"
      >
        <div
          className="products__category"
          id="pizza"
        >
          <h4
            className="products__title h3"
          >
            Пиццы
          </h4>
          <div
            className="products__category_content"
          />
        </div>
        <div
          className="products__category"
          id="pasta"
        >
          <h4
            className="products__title h3"
          >
            Пасты
          </h4>
          <div
            className="products__category_content"
          />
        </div>
        <div
          className="products__category"
          id="soup"
        >
          <h4
            className="products__title h3"
          >
            Супы
          </h4>
          <div
            className="products__category_content"
          />
        </div>
        <div
          className="products__category"
          id="salad"
        >
          <h4
            className="products__title h3"
          >
            Салаты
          </h4>
          <div
            className="products__category_content"
          />
        </div>
        <div
          className="products__category"
          id="snack"
        >
          <h4
            className="products__title h3"
          >
            Закуски
          </h4>
          <div
            className="products__category_content"
          />
        </div>
        <div
          className="products__category"
          id="drink"
        >
          <h4
            className="products__title h3"
          >
            Напитки
          </h4>
          <div
            className="products__category_content"
          />
        </div>
      </div>
    </section>
    <section
      className="our_promo"
    >
      <h2
        className="our_promo__title h2"
      >
        <span>
          Наши 
        </span>
        <span>
          акции
        </span>
      </h2>
      <div
        className="our_promo__content"
      >
        <div
          className="our_promo__promo"
        >
          <img
            alt="promo"
            src="./content/promo_1.jpg"
          />
        </div>
        <div
          className="our_promo__promo"
        >
          <img
            alt="promo"
            src="./content/promo_2.jpg"
          />
        </div>
        <div
          className="our_promo__promo"
        >
          <img
            alt="promo"
            src="./content/promo_3.jpg"
          />
        </div>
        <div
          className="our_promo__promo"
        >
          <img
            alt="promo"
            src="./content/promo_1.jpg"
          />
        </div>
        <div
          className="our_promo__promo"
        >
          <img
            alt="promo"
            src="./content/promo_4.jpg"
          />
        </div>
      </div>
      <button
        className="our_promo__all_promo_btn oval_btn"
      >
        Все акции
      </button>
    </section>
    <section
      className="payment_and_delivery"
    >
      <div
        className="payment_and_delivery__inner"
      >
        <h4
          className="payment_and_delivery__title h3"
        >
          Оплата и доставка
        </h4>
        <div
          className="payment_and_delivery__content"
        >
          <div
            className="payment_and_delivery__benefit"
          >
            <div
              className="payment_and_delivery__benefit_img"
            >
              <img
                alt="benefit"
                src="./icons/shop.png"
              />
            </div>
            <p
              className="payment_and_delivery__benefit_text text"
            >
              Широкий выбор блюд
            </p>
          </div>
          <div
            className="payment_and_delivery__benefit"
          >
            <div
              className="payment_and_delivery__benefit_img"
            >
              <img
                alt="benefit"
                src="./icons/car_with_paper.png"
              />
            </div>
            <p
              className="payment_and_delivery__benefit_text text"
            >
              Полная информация о заказе
            </p>
          </div>
          <div
            className="payment_and_delivery__benefit"
          >
            <div
              className="payment_and_delivery__benefit_img"
            >
              <img
                alt="benefit"
                src="./icons/check.png"
              />
            </div>
            <p
              className="payment_and_delivery__benefit_text text"
            >
              Удобное составление и быстрое приготовление заказа
            </p>
          </div>
          <div
            className="payment_and_delivery__benefit"
          >
            <div
              className="payment_and_delivery__benefit_img"
            >
              <img
                alt="benefit"
                src="./icons/car_with_time.png"
              />
            </div>
            <p
              className="payment_and_delivery__benefit_text text"
            >
              Доставка в течении 1 часа
            </p>
          </div>
        </div>
        <iframe
          allowFullScreen=""
          className="payment_and_delivery__map map"
          loading="lazy"
          referrerPolicy="no-referrer-when-downgrade"
          src="https://www.google.com/maps/embed?pb=!1m17!1m12!1m3!1d2250.5728539025727!2d37.47757307673972!3d55.66163797305065!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m2!1m1!2zNTXCsDM5JzQxLjkiTiAzN8KwMjgnNDguNSJF!5e0!3m2!1sen!2sru!4v1721155631161!5m2!1sen!2sru"
        />
      </div>
    </section>
  </main>,
  <footer
    className="footer"
  >
    <div
      className="footer__left_side"
    >
      <div
        className="footer__logo"
      >
        <a
          className="logo"
          href="/"
          onClick={[Function]}
        >
          <img
            alt="logo"
            src="./icons/logo_1.png"
          />
        </a>
      </div>
      <div
        className="footer__info"
      >
        <a
          className="footer__info_link link"
          href="#"
        >
          Калорийность и состав
        </a>
        <a
          className="footer__info_link link"
          href="#"
        >
          Правовая информация
        </a>
      </div>
      <nav
        className="footer__nav"
      >
        <h6
          className="footer__nav_title h6"
        >
          Мы в соцсетях
        </h6>
        <ul
          className="footer__nav_list"
        >
          <li
            className="footer__nav_item link_in_list__item"
          >
            <a
              className="footer__nav_link link"
            >
              YouTube
            </a>
          </li>
          <li
            className="footer__nav_item link_in_list__item"
          >
            <a
              className="footer__nav_link link"
            >
              Facebook
            </a>
          </li>
          <li
            className="footer__nav_item link_in_list__item"
          >
            <a
              className="footer__nav_link link"
            >
              Instagram
            </a>
          </li>
          <li
            className="footer__nav_item link_in_list__item"
          >
            <a
              className="footer__nav_link link"
            >
              ВКонтакте
            </a>
          </li>
          <li
            className="footer__nav_item"
          >
            <address
              className="footer__nav_address"
            >
              <span>
                 Москва ул. Проспект 
              </span>
              <span>
                 Вернадского 86В 
              </span>
            </address>
          </li>
        </ul>
      </nav>
      <div
        className="footer__copyright"
      >
        <span>
          YaBao Все права защищены © 2021
        </span>
        <div
          className="footer__copyright_imgs"
        >
          <div
            className="footer__copyright_img"
          >
            <img
              alt="visa"
              src="./icons/visa.png"
            />
          </div>
          <div
            className="footer__copyright_img"
          >
            <img
              alt="paypal"
              src="./icons/paypal.png"
            />
          </div>
          <div
            className="footer__copyright_img"
          >
            <img
              alt="mastercard"
              src="./icons/mastercard.png"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="footer__right_side"
    >
      <nav
        className="footer__contacts"
      >
        <h6
          className="footer__contacts_title h6"
        >
          Остались вопросы? А мы всегда на связи:
        </h6>
        <ul
          className="footer__contacts_list"
        >
          <li
            className="footer__contacts_item link_in_list__item"
          >
            <a
              className="footer__contacts_link_img link"
            >
              <img
                alt="viber"
                src="./icons/viber.png"
              />
            </a>
          </li>
          <li
            className="footer__contacts_item link_in_list__item"
          >
            <a
              className="footer__contacts_link_img link"
            >
              <img
                alt="skype"
                src="./icons/skype.png"
              />
            </a>
          </li>
          <li
            className="footer__contacts_item link_in_list__item"
          >
            <a
              className="footer__contacts_link_img link"
            >
              <img
                alt="messenger"
                src="./icons/messenger.png"
              />
            </a>
          </li>
          <li
            className="footer__contacts_item link_in_list__item"
          >
            <a
              className="footer__contacts_link_img link"
            >
              <img
                alt="telegram"
                src="./icons/telegram.png"
              />
            </a>
          </li>
          <li
            className="footer__contacts_item link_in_list__item"
          >
            <a
              className="footer__contacts_link_img link"
            >
              <img
                alt="facebook"
                src="./icons/facebook.png"
              />
            </a>
          </li>
          <li
            className="footer__contacts_item link_in_list__item"
          >
            <a
              className="footer__contacts_link_img link"
            >
              <img
                alt="vk"
                src="./icons/vk.png"
              />
            </a>
          </li>
          <li
            className="footer__contacts_item link_in_list__item"
          >
            <a
              className="footer__contacts_link link"
            >
              Написать нам
            </a>
          </li>
        </ul>
      </nav>
      <div
        className="footer__phone_info"
      >
        <a
          className="footer__phone_number phone_number"
          href="tel:84993918449"
        >
          8 499 391-84-49
        </a>
        <button
          className="footer__get_call oval_btn"
        >
          Заказать звонок
        </button>
      </div>
    </div>
    <div
      className="footer__bg_img"
    >
      <img
        alt="bg"
        src="./content/logo_2.png"
      />
    </div>
  </footer>,
]
`;
